### PR TITLE
Convenience names for platform data

### PR DESCRIPTION
**Pull request by bjackman**
This lets you parse traces, using platform data, without having to connect to a target.

```
from trace import Trace
from libs.utils.platforms import juno # I don't have a juno board here, but I know my trace was collected from one

trace = Trace(juno, 'trace.dat', ['sched_switch'])
```

### CODE REVIEW

File: libs/utils/platforms/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import json
+import os
+from conf import JsonConf
+
+# Add identifiers for each of the platforms we have JSON descriptors for
+this_dir = os.path.dirname(__file__)
+for file_name in os.listdir(this_dir):
+    name, ext = os.path.splitext(file_name)
+    print name, ext
```
[the remark this answers is not in the document]
Comment (bjackman) [reply libs/utils/platforms/__init__.py]: Oops, sorry thought I pushed a version without this. Will fix.

**Comment by bjackman** [comment]
> Would it be possible to have it generated automatically ? Say, if we give a board name and it isn't in platforms/, we automagically create it for later use. We should be able to get most of the platform data without user interaction.

Yes, I actually copy pasted it from the generated one. I suppose we could automagically do it, but I sort of prefer to know exactly what is in people's source tree, and it isn't very hard to just copy paste it by hand and then add it to the git tree

**Comment by valschneider** [comment]
@bjackman Alright, I agree with you on that. And in any case users can manually do a **te.platform_dump()** for new platforms.

**Comment by bjackman** [comment]
Just remembered about this, I've pushed a version without the debug noise.

**Comment by mdigiorgio** [comment]
LGTM

**Pull request merged**